src/platform_impl/macos/event_loop.rs

```diff
@@ -141,6 +141,8 @@ impl<T> EventLoop<T> {
         F: 'static + FnMut(Event<'_, T>, &RootWindowTarget<T>, &mut ControlFlow),
     {
         self.run_return(callback);
+        // Drop the callback before exiting
+        self._callback = None;
```

Comment:
```suggestion
        drop(self._callback.take());
```
This should invalidate the need for a comment